genePred/compare.go

```diff
@@ -17,7 +17,7 @@ func AllAreEqual(a []*GenePred, b []*GenePred) bool {
 	return true
 }
 
-func Equal(a *GenePred, b *GenePred) bool {
+func Equal(a GenePred, b GenePred) bool {
 	if strings.Compare(a.Id, b.Id) != 0 {
 		log.Print("1")
```

Comment:
These look like they are leftover from debugging?  Or maybe they are meant as diagnostics for when they don't match?  In that second case, maybe this should be in the _test.go file since it is only meant to be used in testing?

Reply:
my thought was that this could be used for comparison purposes in general, so if you wanted to compare entries directly. It's not necessarily useful right at this moment, but i think it could be later on. I think I'm going to return a []int with the bool so that this can be used downstream and replace the prints. Maybe a 1 in every position where the fields dont match and a 0 in the positions they do?